sdks/java/testing/nexmark/src/main/java/org/apache/beam/sdk/nexmark/Main.java

```diff
@@ -255,7 +255,9 @@ private static String produceMeasurement(
     schemaResults.put("timestamp", timestamp);
     schemaResults.put("runner", runner);
     schemaResults.put("measurement", measurement);
-    schemaResults.put("runtimeMs", runtimeMs);
+    schemaResults.put(
+        "runtimeMs",
+        runtimeMs + "i"); // we need to add 'i' suffix since influx default type is float
```

Comment:
What about something like this?
`
By default, InfluxDB treats all values as floats. We need to add 'i' suffix to interpret the value as an integer.
`

Reply:
Fine for me, updated